viewer/packages/utilities/src/events/PointerEvents.ts

```diff
@@ -7,7 +7,7 @@
  * @beta
  */
 export class PointerEvents {
-  public onHover(_event: PointerEvent): void {}
```

Comment:
Why Hover event is changed to `MouseEvent`?
`PointerEvent` basically inherits `MouseEvent`, if `MouseEvent` is used than Touch/Pen will not work

Reply:
I know. But hover is never used on Touch. The position of your finders are unknown anyway. So I wanted to synchronize it. sometimes it uses Mouse and sometime Pointer.

Reply:
I will update the API doc when you have approved.